WEEK6/app/src/main/java/com/example/week6/BlankFragment.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.week6
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.activity.OnBackPressedCallback
+import androidx.activity.addCallback
+import androidx.fragment.app.setFragmentResult
+import com.example.week6.databinding.FragmentBlankBinding
+import kotlin.random.Random
+
+
+class BlankFragment : Fragment() {
+    lateinit var binding: FragmentBlankBinding
+    var rand_max_num = 0
```

Comment:
```suggestion
    var randomMaxNum = 0
```
요렇게 하는게 코드 통일성이 더 있을 것 같네용